pkg/flex/mounter.go

```diff
@@ -53,9 +53,40 @@ func (m *Mounter) Mount(targetPath string, specString string) *Response {
 		return NewFailResponse("Failed to create v3io FUSE container", err)
 	}
 
+	if err := m.createDirs(spec, targetPath); err != nil {
+		return NewFailResponse("Failed to create folders", err)
+	}
+
 	return NewSuccessResponse("Successfully mounted")
 }
 
+func (m *Mounter) createDirs(spec Spec, targetPath string) error {
+	var dirsToCreate []DirToCreate
+	if err := json.Unmarshal([]byte(spec.DirsToCreate), &dirsToCreate); err != nil {
+		return fmt.Errorf("Failed to parse dirsToCreate [%s]: %s", spec.DirsToCreate, err.Error())
+	} else {
+		for _, dir := range dirsToCreate {
+			if strings.HasPrefix(dir.Name, "/") {
+				return fmt.Errorf("Only creation of relative path is supported (%s)", dir.Name)
+			}
+			dirToCreate := fmt.Sprintf("%s/%s", targetPath, dir.Name)
+			if _, err := os.Stat(dirToCreate); err != nil {
```

Comment:
go frowns upon if ... else for error handling. simplify by returning early as such:
```
_, err = stat()
if err == nil {
	// log that it exists and skip
	continue
}

if err != os.IsNotExist(err) {
	return errors.Wrap(err)
}

// mkdir
```